tests/integration/spe1_st/test_load_data.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+
+
+@pytest.mark.spe1
+def test_read_experiments(requests_get):
+    experiments = requests_get("experiments").json()
+    assert len(experiments) == 1
+
+
+@pytest.mark.spe1
```

Comment:
Is it not complaining that `spe1` was not added as a marker?; ie. 
```
[tool.pytest.ini_options]
markers =
    spe1_st
```
in pyproject.toml?

Reply:
if i'm not mistaken that is just some linting fluff which is not relevant - i've think i've seen it on others markers as well. If you run this by yourself it is working. But i might be wrong ?

Reply:
Just getting a warning:
```
PytestUnknownMarkWarning: Unknown pytest.mark.spe1 - is this a typo?  You can register custom marks to avoid this warning
```
So I guess it's fine.